server/src/main/java/io/pravega/schemaregistry/storage/impl/schemas/PravegaKVSchemas.java

```diff
@@ -0,0 +1,192 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.schemaregistry.storage.impl.schemas;
+
+import io.pravega.common.Exceptions;
+import io.pravega.common.concurrent.Futures;
+import io.pravega.schemaregistry.common.HashUtil;
+import io.pravega.schemaregistry.contract.data.SchemaInfo;
+import io.pravega.schemaregistry.storage.StoreExceptions;
+import io.pravega.schemaregistry.storage.client.TableStore;
+import io.pravega.schemaregistry.storage.client.Version;
+import io.pravega.schemaregistry.storage.impl.group.records.NamespaceAndGroup;
+
+import java.util.AbstractMap;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+
+import static io.pravega.schemaregistry.storage.impl.schemas.SchemaRecords.SchemaFingerprintKey;
+import static io.pravega.schemaregistry.storage.impl.schemas.SchemaRecords.SchemaGroupsKey;
+import static io.pravega.schemaregistry.storage.impl.schemas.SchemaRecords.SchemaGroupsList;
+import static io.pravega.schemaregistry.storage.impl.schemas.SchemaRecords.SchemaRecord;
+import static io.pravega.schemaregistry.storage.impl.schemas.SchemaRecords.SchemaIdKey;
+import static io.pravega.schemaregistry.storage.impl.schemas.SchemaRecords.SchemaIdList;
+import static io.pravega.schemaregistry.storage.impl.schemas.SchemaRecords.KeySerializer;
+import static io.pravega.schemaregistry.storage.impl.schemas.SchemaRecords.fromBytes;
+
+public class PravegaKVSchemas implements Schemas<Version> {
```

Comment:
Rename to `PravegaKeyValueSchemas` or the like. Also add JavaDocs.

Reply:
javadocs are in the interface